.github/workflows/author_approval.yml

```
@@ -29,7 +29,7 @@ jobs:
           COMMENTER: ${{ github.event.comment.user.login }}
         shell: julia --compile=min --optimize=0 --color=yes {0}
         run: |
-          m = match(r"Created by: @([A-Za-z0-9]*+)(?:$|\n)", ENV["PR_BODY"])
+          m = match(r"Created by: @([^\s]+)", ENV["PR_BODY"])
```

`[^\s]` should be equivalent to `[\S]`, right?

```suggestion
          m = match(r"Created by: @([\S]+)", ENV["PR_BODY"])
```

---

yeah, true! And then we can drop the `[]`:
```suggestion
          m = match(r"Created by: @(\S+)", ENV["PR_BODY"])
```
e.g.
```julia
julia> match(r"Created by: @(\S+)", "Created by: @hello")
RegexMatch("Created by: @hello", 1="hello")
```